Libraries/LibBareMetal/IO.h

```diff
@@ -64,7 +64,7 @@ inline u32 in32(u16 port)
     return value;
 }
 
-inline void repeated_in16(u16 port, u8* buffer, int buffer_size)
+inline void repeated_in16(u16 port, u16* buffer, int buffer_size)
```

Comment:
This seems like it would break existing code. If that's the case, it belongs in the following commit since it won't even build without this one.